homeassistant/auth/mfa_modules/__init__.py

```diff
@@ -56,13 +57,24 @@ def name(self) -> str:
         """Return the name of the auth module."""
         return self.config.get(CONF_NAME, self.DEFAULT_TITLE)
 
+    @property
+    def multiple(self) -> bool:
```

Comment:
If my understanding is correct, you are implementing a feature to allow one MFA module stores multiple keys, for example, you can have 10 fingerprints keys for a fingerprints MFA module.

To simplify the code review process, it would be better if you can separate this feature to another pull request, I don't think multiple keys is a MUST HAVE feature for webauthn, isn't it?

Reply:
I agree, let's not add support multiple keys in the same PR as adding a new module.

Reply:
Late to the party, but this is in fact a CORE feature of webauthn / u2f

You should always allow a single user to register multiple keys.